examples/connectivity/plot_adhd_tangent.py

```diff
@@ -0,0 +1,93 @@
+"""
+Comparing different connectivity measures
+=========================================
+
+This example compares different measures of functional connectivity between
+regions of interest.
+
+"""
+
+# Fetch dataset
+import nilearn.datasets
+atlas = nilearn.datasets.fetch_atlas_msdl()
+dataset = nilearn.datasets.fetch_adhd()
```

Comment:
I speeded up the example by replacing the StratifiedShuffleSplit by StratifiedKFold. I can decrease the subjects number to 24, the classification results remain OK. But does it make sense to run a classification on 24 subjects?